code/modules/power/power_exporter.dm

```diff
@@ -0,0 +1,153 @@
+/obj/machinery/power/exporter
+	name = "power exporter"
+	desc = "It exports power for points, points get rewards.</span>"
+	icon = 'icons/obj/machines/dominator.dmi'
+	icon_state = "dominator"
+	density = 1
+	anchored = 0
+	verb_say = "states"
+	var/drain_rate = 0	// amount of power to drain per tick
+	var/power_drained = 0 		// has drained this much power
+	var/rating = ""
+	var/active = 0
+	var/rewarded = 0
+
+/obj/machinery/power/exporter/examine(mob/user)
+	..()
+	if(!active)
+		user << "<span class='notice'>The exporter seems to be offline.</span>"
+	else
+		user << "<span class='notice'>The [src] is exporting [drain_rate] kilowatts of power, it has consumed [power_drained] kilowatts so far.</span>"
+
+
+/obj/item/clothing/gloves/krav_maga/engi // a short lived meme
+	name = "fists of the singulo"
+	desc = "You have spent so much time managing power that your fists have become one with the powernet."
+	icon_state = "yellow"
+	item_state = "yellow"
+	item_color="yellow"
+	permeability_coefficient = 0.05
+	cold_protection = HANDS
+	min_cold_protection_temperature = GLOVES_MIN_TEMP_PROTECT
+	heat_protection = HANDS
+	max_heat_protection_temperature = GLOVES_MAX_TEMP_PROTECT
+	resistance_flags = 0
+
+/obj/machinery/power/exporter/attackby(obj/item/O, mob/user, params)
+	if(!active)
+		if(istype(O, /obj/item/weapon/wrench))
+			if(!anchored && !isinspace())
+				connect_to_network()
+				user << "<span class='notice'>You secure the [src] to the floor.</span>"
+				anchored = 1
+			else if(anchored)
+				disconnect_from_network()
+				user << "<span class='notice'>You unsecure and disconnect the [src].</span>"
+				anchored = 0
+			playsound(src.loc, 'sound/items/Deconstruct.ogg', 50, 1)
+			return
+	return ..()
+
+/obj/machinery/power/exporter/attack_hand(mob/user)
+	..()
+	if (!anchored)
+		user << "<span class='warning'>This device must be anchored by a wrench!</span>"
+		return
+	if (power_drained >= 4000000 && !rewarded) // 4 billion watts

```

Comment:
And pray tell what standards are met when you take a number that gets used once and turn it into a DEFINE.

I'm not seeing your point besides trying to give me more hoops to jump through, if you're that passionate about coding at the highest standards just close your eyes and pick a random file and I guarantee you'll find some of actual substance to correct. Until then we're at 270 comments, this PR is not lacking in effort, but it's just not helpful to have to stop and argue about whether or not a single number in a prototype admin-only system is worth turning into a definition or not.